libs/SalesforceSDK/src/com/salesforce/androidsdk/push/PushMessaging.kt

```diff
@@ -160,10 +160,9 @@ object PushMessaging {
                 FirebaseInstallations.getInstance(firebaseApp).delete()
                 firebaseApp.delete()
             }
+
             unregisterSFDCPush(context, account)
         }
-
-        unregisterSFDCPush(context, account)
```

Comment:
This seems like an oversight.  There is no reason for `unregisterSFDCPush` to be called twice here (once even if it wasn't registered 👀).